### ISSUE TITLE
Loss function setting

### ISSUE THREAD

**Issue by chengjing11** [Dong-Huo/ASPDC#4]
Sorry for bothering you. My PSNR after reproduction is only 30.48, I think I didn't change the code correctly, may I ask how the loss function is set at the very beginning of training?

**Comment by Dong-Huo** [comment]
Have you tried the checkpoint that I released?

**Comment by chengjing11** [comment]
Sorry for such a late reply, I just tested the checkpoint that you released and found that the results were consistent with your paper, but my own training was incorrect. I suspect that the forword and backword functions are wrong, can you please see if this is the case? extremely grateful！
def forward(self):
self.deblurred_rgb, _ = self.deblurring_net.forward(self.HR_blurry_patch)
###### Used in training ######
self.reblurred_rgb = self.blurring_net.forward(self.HR_blurry_patch, self.HR_sharp_patch)
###### Used in fine-tuning ######
self.reblurred_rgb = self.blurring_net.forward(self.HR_blurry_patch, self.deblurred_rgb)
def backward(self):
# Second, G(A) = B
self.loss_deblurring_pixel = self.perceptual_loss.get_loss(self.deblurred_rgb, self.HR_sharp_patch)
self.loss_blurring_pixel = self.perceptual_loss.get_loss(self.HR_blurry_patch, self.reblurred_rgb)
self.loss = self.loss_deblurring_pixel * self.cfg.lambda_A + self.loss_blurring_pixel * self.cfg.lambda_B
self.loss.backward()

**Comment by Dong-Huo** [comment]
I am not sure if you train these two losses from scratch. I trained the deblurring and blurring separately and then fine-tune them together.

**Comment by chengjing11** [comment]
Oh, I misunderstood, I was training together to cause this, thank you so much for your patience!
